libs/test-utils/src/expect_print.ts

```diff
@@ -0,0 +1,144 @@
+import { render, RenderResult } from '@testing-library/react';
+import {
+  ElementWithCallback,
+  Optional,
+  PrintOptions,
+} from '@votingworks/types';
+import { advancePromises } from './advance_timers';
+import { assert } from './assert';
+
+export class ExpectPrintError extends Error {}
+
+export type InspectPrintFunction = (
+  renderResult: RenderResult,
+  printOptions?: PrintOptions
+) => void;
+
+let lastPrintedElement: Optional<JSX.Element>;
+let lastPrintedElementWithCallback: Optional<ElementWithCallback>;
+let lastPrintOptions: Optional<PrintOptions>;
```

Comment:
There is a reason, although it may not be the right one.

If you use jest mock call tracking (which I started with) you're using the global `jest` object in the outer scope of a module. `jest` is then an actual or peer dependency and not a dev dependency of `test-utils`. As a result, a couple of the integration testing suites that rely on `test-utils` was failing because `jest` was not defined. We could jest (get it) make `jest` a real dependency, which would probably be fine. But it felt strange if it was avoidable. And since we're clearing the queue, we're not really using any features beyond what we have here.

Here is a comment I made before I took this route: https://github.com/votingworks/vxsuite/pull/2730#discussion_r1001335733.

Reply:
The thing to watch out for here is that it should likely be a `peerDependencies` entry and not `dependencies`, otherwise we could end up with different versions (possibly conflicting ones) of Jest. This would likely only matter for type checking purposes since we'd never directly `import` Jest, but still.

Reply:
@jonahkagan, `test-utils` is also a providing functions to be used with _cypress_ tests.

Reply:
Oh I see. I didn't realize that's what you meant by "integration tests" in your original comment. Sorry about that. I don't feel strongly about this, so if it feels like more trouble than it's worth to figure out, I'm fine with keeping it as is

Reply:
"integration tests" is vague, oops. I don't think it's much trouble to figure out, but I don't see what we have to gain by using jest functions.